extensions/gc/tasks/GarbageCollectorTask.js

```diff
@@ -184,7 +184,25 @@ class GarbageCollectorTask extends BackbeatTask {
         });
     }
 
-    _deleteArchivedSourceData(entry, log, done) {
+    _executeDeleteData(entry, log, done) {
+        const logFields = {
+            bucket: entry.getAttribute('source.bucket'),
+            objectKey: entry.getAttribute('source.objectKey'),
+            storageClass: entry.getAttribute('source.storageClass'),
+            ruleType: entry.getContextAttribute('ruleType'),
+        };
+
+        this.retry({
+            actionDesc: 'execute delete data',
+            logFields,
+            maxRetries: 2,
```

Comment:
No need to specify `maxRetries`, that way it uses the value set in the config and which is configured by ZKOP.

Reply:
done here : https://github.com/scality/backbeat/pull/2520/commits/1e91524b7b2b2b6de6067258f1b53101da49daec#diff-7beaf0ffcd262dad5a78c1b25cb7d54b9ac577441b372167f96d8c182be8fcf7R20-R24